elastic/kibana
# make defaultRoute accessible in NP Config

## Summary

Closes: https://github.com/elastic/kibana/issues/52076
I didn't put back docs removed in https://github.com/elastic/kibana/pull/44678/

how to test:
- set "/app/something" in kibana.yml and open `/` in browser. expected result: you land on "/app/something" page

### Checklist

Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.


### For maintainers
